services/hanke-service/Dockerfile-local

```
@@ -1,21 +1,21 @@
-FROM eclipse-temurin:17 as builder
-USER root
+FROM eclipse-temurin:17 as build
+WORKDIR /workspace/app
 
-# Copy gradle to avoid redownloading it
-COPY *.gradle gradle.* gradlew /builder/
-COPY gradle /builder/gradle
-WORKDIR /builder
-RUN ./gradlew --version
-RUN java -version
+COPY gradlew settings.gradle.kts ./
+COPY gradle/ ./gradle/
+COPY services/hanke-service/build.gradle.kts ./services/hanke-service/
+COPY services/hanke-service/src ./services/hanke-service/src
```

This only needs main sources, since it's not running tests. Changing this would mean that changes in test sources don't affect Docker's caching.
```suggestion
COPY services/hanke-service/src/main ./services/hanke-service/src/main
```